plonky2x/src/frontend/eth/mpt/builder.rs

```diff
@@ -51,7 +51,26 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
             _phantom: PhantomData,
         };
         self.add_simple_generator(generator.clone());
-        generator.output
+
```

Comment:
nit: maybe we should have 2 functions `lt` and `le` and basically `le` will call `lt` and then at the end do 
```
        let eq = self.is_equal(lhs, rhs);
        let lte = self.or(lt, eq);
```

Reply:
fixed